### PR TITLE
feat(org-stats): Remove feature flag

### PR DESCRIPTION
**Pull request by leedongwei**
~Renamed to `OrganizationStatsV2` to maintain the pattern of `EventsV2`, `DashboardsV2`~ 

On the frontend, removing all traces of OrganizationStatsV1 so we won't need to change the filepath ever again on `getsentry`. 

The backend will still have V1/V2. Keeping it around as there are a non-trivial amount of orgs hitting the V1 endpoint. No real business/engineering reason to force them off V1 atm, so we'll let them slowly migrate (which should happen if they want to record their usage of transactions/attachments).

### CODE REVIEW

**Comment by markstory** [comment]
> Renamed to OrganizationStatsV2 to maintain the pattern of EventsV2, DashboardsV2 😆

I don't think this is a pattern we should continue. Ideally those directories would be renamed as well.

File: tests/js/spec/views/organizationStatsV2/index.spec.jsx [hunk 1]
```diff
@@ -6,10 +6,10 @@ import {initializeOrg} from 'sentry-test/initializeOrg';
 
 import {DEFAULT_RELATIVE_PERIODS, DEFAULT_STATS_PERIOD} from 'app/constants';
 import {DataCategory} from 'app/types';
-import UsageStats from 'app/views/usageStats';
-import {CHART_OPTIONS_DATA_TRANSFORM} from 'app/views/usageStats/usageChart';
+import {OrganizationStats} from 'app/views/organizationStatsV2';
+import {CHART_OPTIONS_DATA_TRANSFORM} from 'app/views/organizationStatsV2/usageChart';
```
Comment (markstory) [inline tests/js/spec/views/organizationStatsV2/index.spec.jsx:10]: Are you planning on renaming this to `organizationStats` once the old code is removed? Keeping a v2 around long term isn't great.

**Review by markstory — APPROVED** [review]

File: tests/js/spec/views/organizationStatsV2/index.spec.jsx (hunk 1 above)
Comment (leedongwei) [reply tests/js/spec/views/organizationStatsV2/index.spec.jsx]: Deleted v1 on frontend, so there won't be v1/v2 there. 

The backend will still have v1/v2 as there are a non-trivial amount of orgs hitting the v1 endpoint.

File: static/app/views/usageStats/types.tsx [hunk 2]
```diff
@@ -1,28 +0,0 @@
-import {SeriesApi} from 'app/types';
```
Comment (leedongwei) [inline static/app/views/usageStats/types.tsx:1]: Not actually deleting this file.

It was moved to `static/app/views/organizationStats/types.tsx` (8 files above this) but git didn't resolve the file movement correctly.

**Review by markstory — COMMENTED** [review]
The renames look good, but they might disrupt getsentry which relies on importing from `sentry/views/usageStats/usageChart/utils` and `sentry/views/usageStats/usageChart`.

**Comment by leedongwei** [comment]
The file paths are updated at https://github.com/getsentry/getsentry/pull/5582/files

**Pull request merged**